text/user.css

```
@@ -120,84 +136,102 @@
 .main-topBar-overlay,
-.main-entityHeader-container > div,
-.main-entityHeader-container + div[style*="background-color"],
-.main-entityHeader-container + div > div[style*="background-color"],
+.main-entityHeader-container>div,
+.main-entityHeader-container+div[style*="background-color"],
+.main-entityHeader-container+div>div[style*="background-color"],
 .main-home-homeHeader,
 .main-nowPlayingView-contextItemInfo::before {
     background-color: transparent !important;
     background-image: none !important;
     background: transparent !important;
```

Stylelint (17.11.0)

[error] Overridden property "background-image" by shorthand "background" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)